libraries/achartengine/achartengine/src/org/achartengine/GraphicalActivity.java

```diff
@@ -15,34 +15,59 @@
  */
 package org.achartengine;
 
-import org.achartengine.chart.AbstractChart;
-
+import android.app.ActionBar;
 import android.app.Activity;
+import android.os.Build;
 import android.os.Bundle;
+import android.view.MenuItem;
 import android.view.Window;
 
+import org.achartengine.chart.AbstractChart;
+
 /**
  * An activity that encapsulates a graphical view of the chart.
  */
 public class GraphicalActivity extends Activity {
-  /** The encapsulated graphical view. */
-  private GraphicalView mView;
-  /** The chart to be drawn. */
-  private AbstractChart mChart;
+    /**
+     * The encapsulated graphical view.
+     */
+    private GraphicalView mView;
+    /**
+     * The chart to be drawn.
+     */
+    private AbstractChart mChart;

```

Comment:
Given the type and name of these fields, the comments don't provide any additional info.